CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## v1.7.6
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* Add possibility to configure native-routing-cidr in helm chart. (Backport PR #12354, Upstream PR #11132, @zbindenren)
+
+**Bugfixes:**
+* avoid having endpoints in 'restoring' state in case the connectivity with the KVStore is not reliable (Backport PR #12333, Upstream PR #12307, @aanm)
```

Comment:
Nit: if we capitalize  "stop" below, then should be consistent here as well.

Reply:
I suppose. Usually I don't bother messing with the changelogs and just encourage the original authors to consider the release notes when they write PR titles & release notes.

Will fix it up anyway.